protocol/messenger_activity_center.go

```diff
@@ -235,6 +235,10 @@ func (m *Messenger) DismissActivityCenterNotifications(ctx context.Context, ids
 	return nil, err
 }
 
+func (m *Messenger) DeleteActivityCenterNotifications(ctx context.Context, ids []types.HexBytes, sync bool) error {
```

Comment:
Should we return response here saying what notifications were deleted? Or we expecting client well delete notifications on the ui by itself?

Reply:
Good question. I went with the current "pattern" of defining an endpoint that's meant to be called in a *fire and forget* style by the clients. At least in status-mobile, a lot of calls to status-go are always assumed to work and the UI is optimistically updated and if an error happens it's only logged or sometimes even ignored :man_shrugging: 

This is something we should definitely improve at some point. But for the way things currently are, I think it's enough to only return nil or an error.